tools/rostopic/src/rostopic/__init__.py

```diff
@@ -1532,19 +1530,15 @@ def _rostopic_cmd_bw(argv=sys.argv):
     from optparse import OptionParser
     parser = OptionParser(usage="usage: %prog bw /topic", prog=NAME)
     parser.add_option("-w", "--window",
-                      dest="window_size", default=None,
+                      dest="window_size", default=-1,
```

Comment:
Why is this change necessary for Python 3 support? `None` seems to be a perfectly reasonable default value.